docs/deploy.md

```diff
@@ -17,19 +19,35 @@ If you would like to run your own instance of this plugin, you can do so by fork
 - Commit Statuses - **Read & Write**
 - Single File - **Read-only**
   - Path: `.github/mergeable.yml`
-- Repository Contents - **Read-Only**  
+- Repository Contents - **Read-Only**
 
 **And subscription to the following events:**
 - [x] Pull request
 - [x] Pull request review comment
 - [x] Pull request review
 - [x] Issues
 
+Make sure to create a private key for the app after it's been registered.
+
 ## Running Locally
 1. Clone the forked repository on to your machine
 2. Globally install smee-client from with npm ```npm install -g smee-client```
-3. Export all the variables required based on the ProBot deployment guide above
-4. Run smee in your terminal by using the ```smee``` command
-5. Run npm start in your local repository
+3. Go to [smee.io](https://smee.io) and create a new webhook OR use the cli by
+   running the `smee` command.
+3. Export all the variables required based on the ProBot deployment guide above:
```

Comment:
@IllegalCactus While GH markdown convert these numbered bullet points accurately, it will be nice to keep the numbers in the correct running order. In this case it should be `4` here.  Alternatively, I'm open to have this all changed to just `1` :-)

Reply:
Woops, good point. Have fixed this.